src/common/Model.ts

```diff
@@ -118,6 +118,10 @@ export interface CreateLocRequest {
     ownerAddress?: string;
     requesterAddress?: string;
     description?: string;
+    firstName?: string;
```

Comment:
For a creation I expect the following payload:
```
{
  "requesterAddress": "5CXLTF2PFBE89tTYsrofGPkSfGTdmW4ciw4vAfgcKhjggRgZ",
  "ownerAddress": "5GrwvaEF5zXb26Fz9rcQpDWS57CtERHpNehXCPcNoHGKutQY",
  "description": "I want to open a case",
  "userIdentity": {
    "firstName": "John",
    "lastName": "Doe",
    "email": "john.doe@logion.network",
    "phoneNumber": "+1234"
  }
}
```
On successful submit, or for fetch, the response is:
```
{
  "id": "3e67427a-d80f-41d7-9c86-75a63b8563a1",
  "requesterAddress": "5CXLTF2PFBE89tTYsrofGPkSfGTdmW4ciw4vAfgcKhjggRgZ",
  "ownerAddress": "5GrwvaEF5zXb26Fz9rcQpDWS57CtERHpNehXCPcNoHGKutQY",
  "description": "I want to open a case",
  "userIdentity": {
    "firstName": "John",
    "lastName": "Doe",
    "email": "john.doe@logion.network",
    "phoneNumber": "+1234"
  },
  "createdOn": "2021-09-23T13:50:33.318Z",
  "status": "REQUESTED"
}
```

Reply:
Done, thanks!